fundamentos/Bloco-03-Introducao-HTML-CSS/dia-4-html-semantico/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <title>Stomatopoda</title>
```

Comment:
Segue sugestão para a tag title, ela é utilizada dentro do head, desta forma: 
```suggestion
<head>
    <title>Stomatopoda</title>
</head>
```